master/buildbot/status/build.py

```diff
@@ -211,21 +212,15 @@ def waitUntilFinished(self):
     def createBuildUserStatus(self, step):
         @defer.inlineCallbacks
         def thd(build_id):
-            for owner in self.owners:
-                # TODO: Change in future to uid from session!
-                # "pyflakes pyflakes@unity3d.com" -> "pyflakes"
-                username = " ".join(owner.split()[:-1])
-                user_id = yield self.master.db.users.getUidByLdapUsername(username)
-                if not user_id:
-                    msg = "Can not find user in database: username: {user}; project: {project};" \
-                          " builder: {builder}"
-                    klog.err_json(msg.format(
-                        user=owner,
-                        project=step.builder.project,
-                        builder=step.builder.friendly_name
-                    ))
-                    continue
-                yield self.master.db.builds.createBuildUser(build_id, user_id, self.finished)
+            if self.user_id:
```

Comment:
minor: It wouldn't be better if we remove `else statement` from this function?
I mean something like this:
```
if not self.user:
    klog...
    return
yield self.master...
```

Reply:
After changes, the `thd` function has one `if` with `yield`